docs/src/pages/components/data-grid/events/CatalogOfEventsNoSnap.js

```diff
@@ -10,33 +23,118 @@ const useStyles = makeStyles({
   },
 });
 
+function escapeHTML(value) {
+  return value.replace(/</g, '&lt;').replace(/>/g, '&gt;');
+}
+
+const EventRow = ({ event }) => {
+  const [open, setOpen] = React.useState(false);
+
+  const example = React.useMemo(() => {
+    const args = ['details // GridCallbackDetails'];
+    if (event.event) {
+      args.unshift(`event,  // ${event.event}`);
+    }
+    if (event.params) {
+      args.unshift(`params, // ${event.params}`);
+    }
+
+    return `
+const onEvent: GridEventListener<GridEvents.${event.name}> =
+ (
+    ${args.join('\n    ')}
+  ) => {...}    
```

Comment:
The indentation is not the same as https://deploy-preview-3022--material-ui-x.netlify.app/components/data-grid/events/#subscribing-to-events